src/envs/behavior.py

```diff
@@ -30,9 +29,8 @@
 
     _BEHAVIOR_IMPORTED = True
     bddl.set_backend("iGibson")  # pylint: disable=no-member
-    if os.path.exists("tmp_behavior_states/"):
-        shutil.rmtree("tmp_behavior_states/")
-    os.makedirs("tmp_behavior_states/")
```

Comment:
why do we not need this anymore?

Reply:
Because these saved states are needed for loading the states to compute the abstracted states during NSRT learning when we `--load_data`.